### ISSUE TITLE
How to determine direction & amount dragged?

### ISSUE THREAD

**Issue by jonjon1123** [asjqkkkk/circle_list#11]
Hi there!

I'm trying to figure out how I can use this library to get the direction the user is dragging (clockwise or counter-clockwise) and the amount they have dragged.  I see there is `onDragUpdate` , but I am not sure how to use the `PolarCoord` from that function to determine the direction and amount dragged.

Any guidance would be greatly appreciated!

**Comment by asjqkkkk** [comment]
Hi, maybe you can use `angle` to determine the direction😄

**Comment by jonjon1123** [comment]
Yeah, I eventually figured that out. Now I am trying to figure out how to determine the velocity of the dragging. I am trying to achieve something similar to this: https://fireship.io/snippets/circular-drag-flutter/. Maybe this lib could just expose `onPanUpdate` of the `GestureDetector`?

**Comment by annalaufey** [comment]
> 

jonjon1123 did you find a solution? I'm looking for something very similar
